argcomplete/_check_module.py

```diff
@@ -12,25 +12,34 @@
 
     def find_spec(name):
         """Minimal implementation as required by `find`."""
-        f, path, _ = find_module(name)
+        try:
+            f, path, _ = find_module(name)
+        except ImportError:
+            return None
         has_location = path is not None
         if f is None:
             return ModuleSpec(None, has_location, [path])
         f.close()
         return ModuleSpec(path, has_location, None)
 
 
+class LocateFailure(RuntimeError):
+    pass
+
+
 def find(name):
     names = name.split('.')
     spec = find_spec(names[0])
+    if spec is None:
+        raise LocateFailure('no module named "{}"'.format(name))
```

Comment:
Nitpicking, but the thing we failed to find is `names[0]` (as opposed to `name`)